nucleus/admin/template/src/main/resources/config/logging.properties

```diff
@@ -74,6 +74,8 @@ fish.payara.enterprise.server.logging.PayaraNotificationFileHandler.rotationLimi
 fish.payara.enterprise.server.logging.PayaraNotificationFileHandler.maxHistoryFiles=0
 fish.payara.enterprise.server.logging.PayaraNotificationFileHandler.file=${com.sun.aas.instanceRoot}/logs/notification.log
 
+fish.payara.deprecated.jsonlogformatter.underscoreprefix=false
```

Comment:
Can you put comments on each use of this saying when it was deprecated from please? Like you have in the JSONLogFormatter class